### PR TITLE
Make Add-on SDK (Jetpack) work on Pale Moon || Part 2

### PR DESCRIPTION
**Pull request by JustOff**
* Provide `gContextMenuContentData` for `sdk/context-menu` (stealed from Basilisk)
* Remove embedded WebExtensions support (bug [1269347](https://bugzilla.mozilla.org/show_bug.cgi?id=1269347))

Tag #120.

### CODE REVIEW

File: toolkit/jetpack/moz.build [hunk 1]
```diff
@@ -128,9 +128,10 @@ EXTRA_JS_MODULES.commonjs += [
     'test.js',
 ]
 
-EXTRA_JS_MODULES.commonjs.sdk += [
-    'sdk/webextension.js',
-]
+if not CONFIG['MC_PALEMOON']:
```
Comment (mattatobin) [inline toolkit/jetpack/moz.build]: Make this `if CONFIG['MOZ_WEBEXTENSIONS']`

File: toolkit/jetpack/sdk/addon/bootstrap.js [hunk 2]
```diff
@@ -134,8 +134,10 @@ Bootstrap.prototype = {
       const main = command === "test" ? "sdk/test/runner" : null;
       const prefsURI = `${baseURI}defaults/preferences/prefs.js`;
 
+#ifndef MC_PALEMOON
```
Comment (mattatobin) [inline toolkit/jetpack/sdk/addon/bootstrap.js]: Make this `#ifdef MOZ_WEBEXTENSIONS`

**Comment by JustOff** [comment]
Does it make sense to add `#ifdef MOZ_JETPACK` around the `gContextMenuContentData` code in `application/palemoon/base/content/content.js` and `application/palemoon/base/content/nsContextMenu.js`?

**Comment by mattatobin** [comment]
I can't make jetpack optional until I can work out what bit of devtools-server is required elsewhere in the build system. Since jetpack is a direct dependency of devtools we have bigger fish to fry ;)

Feel free if you want in a separate PR..

**Pull request merged**